ci/local/build.sh

```diff
@@ -131,7 +131,7 @@ DOCKER_MAJOR=$(docker -v|sed 's/[^[0-9]*\([0-9]*\).*/\1/')
 GPU_OPTS="--gpus device=${NVIDIA_VISIBLE_DEVICES}"
 if [ "$DOCKER_MAJOR" -lt 19 ]
 then
-    GPU_OPTS="--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES='${NVIDIA_VISIBLE_DEVICES}'"
+    GPU_OPTS="--gpus=all -e NVIDIA_VISIBLE_DEVICES='${NVIDIA_VISIBLE_DEVICES}'"
```

Comment:
Any reason this needs to be changed? I think the `--runtime=nvidia` syntax is needed to support users with Docker versions less than 19. This syntax is similar to what we recommend in our DockerHub READMEs (see screenshot, [link](https://hub.docker.com/r/rapidsai/rapidsai-core-dev-nightly))


![image](https://user-images.githubusercontent.com/7400326/136032901-e4174be6-2128-4789-a645-2cf010a3b1a8.png)

Reply:
Thanks for the context! Does `--runtime-nvidia` work with newer versions at all? IIRC docker wasn't happy with it and it's often a pain point for me (need to look back in my notes for what to change it to).

Reply:
I believe it should. I have the latest version of Docker installed on my machine and I use `--runtime=nvidia` regularly